analyzers/src/SonarAnalyzer.CSharp/Rules/DeadStores.cs

```diff
@@ -224,14 +190,10 @@ private void ProcessOpAssignment(SyntaxNode instruction, HashSet<SyntaxNode> ass
             {
                 var assignment = (AssignmentExpressionSyntax)instruction;
                 var left = assignment.Left.RemoveParentheses();
-                if (left.IsKind(SyntaxKind.IdentifierName))
+                if (left.IsKind(SyntaxKind.IdentifierName)
```

Comment:
The condition of this if is repeated 3 times, it can be extracted to a method.

Reply:
Nice, except the coverage went under 95 🤦‍♂️ 😆